src/display.cpp

```diff
@@ -1695,6 +1698,14 @@ void display::draw_wrap(bool update, bool force)
 		update_display();
 
 		frametimes_.push_back(SDL_GetTicks() - last_frame_finished_);
+		fps_counter_++;
+		const std::chrono::seconds current_second = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch());
```

Comment:
Please use `std::chrono::steady_clock` for this. `system_clock` is intended only for display.